scripts/toaster_configuration

```diff
@@ -151,12 +154,15 @@ EOF
 
 for layer in $(echo "$LAYER_PATHS"); do
    name=
-   layername=$(basename "$layer")
-   if [ "$layername" = "meta" ]; then
-        continue
-   fi
-   collectLayers="$collectLayers,$layername"
-   dirpath=$(basename "$layer")
+   layername=$(grep "BBFILE_COLLECTIONS *+= *\"" ${layer}/conf/layer.conf | cut -d '"' -f2)
+   case $layername in
+       core) continue
+             ;;
+       meta-*)
+             ;;
+       *) layername="meta-${layername}"
```

Comment:
More duplicated code. Also, what's the point of artificially adding 'meta-'? Not all layers include that at all, either in the path on disk or the layer name internally.